script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -1457,12 +1510,16 @@ You have something to say, don't you? Go ahead and get whatever complaints or gr
 [sha:9a9bfec1ecf7b950c8ce86510572cc9b2f40c69f]{
 -- Page 193, Offset 6101.
 -- 「そりゃあ言いたいコトはいっぱいあるけど……あなた、変わった人ね。吸血鬼を前にして、なに、その態度？」
-"There's a lot I want to say, but... you're a strange one. What kind of attitude is that, especially in front of a vampire?"
+"Well yeah, there's a lot a want to say, but... you're a strange one.
+ You're face to face with a vampire, what's with that attitude?"
 }
 [sha:1299c9b98f96632d781109ba9c88b239759f729d]{
 -- Page 193, Offset 6102.
 -- 「悪かったな、必死に居直ってるんだよ。……そうでもしないと、自分で自分を殺しかねない」
-"I'm sorry, I'm trying really hard not to lose my mind... if I did, I might even kill myself."
+// 居直る is like 'go on the offensive', I assume this is the same idea as
+// 6078-9 where he's getting angry so he doesn't have to dwell on what he did?
+"Sorry, I'm pretty hopped up right now...
```

Comment:
not a fan of "hopped up". It sounds like he's on drugs

Reply:
Riled up?

Reply:
works for me